srcpkgs/nim/template

```diff
@@ -15,9 +15,18 @@ homepage="https://nim-lang.org/"
 distfiles="https://github.com/nim-lang/Nim/archive/v${version}.tar.gz
  https://github.com/nim-lang/csources/archive/v${_cversion}.tar.gz>csources-${_cversion}.tar.gz
  https://github.com/nim-lang/nimble/archive/v${_nimbleversion}.tar.gz>nimble-${_nimbleversion}.tar.gz"
-checksum="dbd5f1784ab121071aa7c58bb6b361cca79d12ac7524c761e27200e0c946e359
+checksum="3918ff1834888cd3feec5002931d4052a354994a0355bf1ae950f2f17ba81417
  5e6fd15d90df1a8cb7614c4ffc70aa8c4198cd854d7742016202b96dd0228d3c
- 63e0dbfa5c33f80d6f8f8156a609e11497ac5390b64dc8d68f5126c3bb8d5429"
+ 0b88d91a450f31641f85379f2d76afb0a013cf783e62144a6534525b9cb1cbac"
+
+# Git is needed to build on x86_64* and i686* for some reason,
+# while on other architectures Nim builds normally...
+# https://api.travis-ci.org/v3/job/736593571/log.txt
+case "$XBPS_TARGET_MACHINE" in
+	x86_64*|i686*)
+	hostmakedepends+=" git"
+	;;
+esac
```

Comment:
This is probably a case of cross compiling vs native build...

Would be interesting to find out what changes.

Reply:
If you remove the `hostmakedepends+=git` line when natively compiling the compilation doesn't succeed for some reason but by looking at the build log it uses git to download some extra modules (maybe the Fusion libraries contained in Nim).

Strangely enough cross-compilation doesn't require git as a host dependency when building the compiler from source :thinking: